spec/controllers/key_registration.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
```

Comment:
This file should probably be called `key_registration_spec.rb` to match the usual conventions.